lab-khalid/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const greeting = require('../lab-khalid/lib/greet.js');
+console.log('working');
+greeting.hello(process.argv[2]);
```

Comment:
Great use of process.argv and passing it into the greeting module's hello method.